### PR TITLE
Remove MSVC workaround: wmain link error

### PR DESCRIPTION
**Pull request by kuzkry**
Two commits here:
298a40f0 - removing workaround to the bug that has been patched
a909becd - just me secretly trying to smuggle some unification to both main files

### CODE REVIEW

**Comment by rogeeff** [comment]
Why we do not need this workaround anymore?

**Comment by kuzkry** [comment]
Based on [Microsoft C++ language conformance table](https://docs.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=vs-2019) you can check that the earliest implementation of C++11 is in VS 2015.
The comment in code has a link to a bug that has been confirmed as fixed in VS 2010.

**Comment by rogeeff** [comment]
For copy-pasting: “Thank you, we have started internal review. Please don’t push any more changes into this PR as they might be overwritten. (282585468)

File: googlemock/src/gmock_main.cc [hunk 1]
```diff
@@ -48,21 +48,14 @@ void loop() { RUN_ALL_TESTS(); }
...
 #else
-
-// MS C++ compiler/linker has a bug on Windows (not on Windows CE), which
-// causes a link error when _tmain is defined in a static library and UNICODE
-// is enabled. For this reason instead of _tmain, main function is used on
-// Windows. See the following link to track the current status of this bug:
-// https://web.archive.org/web/20170912203238/connect.microsoft.com/VisualStudio/feedback/details/394464/wmain-link-error-in-the-static-library
-// // NOLINT
-#if GTEST_OS_WINDOWS_MOBILE
+#if __MSC_VER
```
Comment (chfast) [inline googlemock/src/gmock_main.cc]: I think it should have been `_MSC_VER`.

Comment (kuzkry) [reply googlemock/src/gmock_main.cc]: I don't know how you managed to find it :astonished:

**Pull request merged**